docs/agent-k8s-install-helm.md

```diff
@@ -15,8 +15,7 @@ to install the Smart Agent. To learn more, see
 * Terminal or a similar command-line interface application
 * Helm:
 
-  - The installation chart for the Smart Agent is compatible with Helm version 2 or Helm version 3.
-  - To learn more, see the [Helm site](https://helm.sh/).
+  - The installation chart for the Smart Agent is compatible with Helm version 2 or Helm version 3. Helm version 2 is [deprecated](https://helm.sh/blog/helm-v2-deprecation-timeline/). To learn more, see the [Helm site](https://helm.sh/).
```

Comment:
```suggestion
  - The installation chart for the Smart Agent is compatible with Helm version 3. Helm version 2 is [deprecated](https://helm.sh/blog/helm-v2-deprecation-timeline/). To learn more, see the [Helm site](https://helm.sh/).
```

Reply:
@tmorrisonsfx If we're including the `--generate-name` option in the install commands, we should drop Helm v2 compatibility.